src/seedu/addressbook/AddressBook.java

```diff
@@ -1212,6 +1212,19 @@ private static String getUsageInfoForExitCommand() {
 	 */
 
 	/**
+	 * Removes prefix from parameter string
+	 * 
+	 * @param fullstring
+	 *            parameter string
+	 * @param prefix
+	 *            prefix to remove from parameter string
+	 */
+
+	private static String removePrefix(String fullstring, String prefix) {
+		return fullstring.replaceFirst("^" + prefix, "");
```

Comment:
indentation is off here.
please use 4 spaces for indentation as per the coding guidelines.
Tip: you can get eclipse to do that for you. check out how!